ci-operator/config/openshift/multiarch-manager-operator/openshift-multiarch-manager-operator-main.yaml

```diff
@@ -83,24 +93,38 @@ tests:
     - ref: ipi-install-heterogeneous
     - as: test
       cli: latest
-      commands: |
-        #!/bin/bash
-        mkdir -p /tmp/bin
-        export PATH=/tmp/bin:$PATH
-        ln -s $(which oc) /tmp/bin/kubectl
-        oc create namespace openshift-multiarch-manager-operator
-        oc annotate namespace openshift-multiarch-manager-operator \
-          scheduler.alpha.kubernetes.io/node-selector="kubernetes.io/arch=amd64"
-        export NO_DOCKER=1
-        # TODD: The following should change to use the operator-sdk and OLM
-        make deploy IMG=${OPERATOR_IMAGE}
-        oc wait deployments -n openshift-multiarch-manager-operator \
-          -l app.kubernetes.io/part-of=multiarch-manager-operator \
-          --for=condition=Available=True
-        oc wait pods -n openshift-multiarch-manager-operator \
-          -l control-plane=controller-manager \
-          --for=condition=Ready=True
-        make e2e
+      commands: USE_OLM=true ./hack/deploy-and-e2e.sh
```

Comment:
error for this job is it can't find `./hack/deploy-and-e2e.sh`

Reply:
because this is running through operator-sdk it needs to be something like
```
oc create namespace openshift-multiarch-manager-operator
operator-sdk run bundle  "$OO_BUNDLE"
oc wait --for condition=Available -n openshift-multiarch-manager-operator deployment multiarch-manager-operator
```

Reply:
because of the `from: operator-sdk`, i'm not sure this container can access the multiarch-manager-operator hack scripts

Reply:
i think we're good now... the cluster installation failed tho.

I'l also add a promotion stanza.